src/uwtools/tests/drivers/test_driver.py

```diff
@@ -397,6 +400,22 @@ def test_Driver_namelist_schema_default_disable(driverobj):
         assert driverobj.namelist_schema() == {"type": "object"}
 
 
+def test_Driver_output(config):
+    driverobj = ConcreteDriverTimeInvariant(config)
+    assert driverobj.output == {"foo": "/path/to/foo", "bar": ["/path/to/bar1", "/path/to/bar2"]}
+
+
+@mark.parametrize("cls", [ConcreteDriverCycleBased, ConcreteDriverCycleLeadtimeBased])
+def test_Driver_output_not_implemented(cls, config):
+    kwargs = {"config": config, "cycle": dt.datetime(2024, 10, 22, 12)}
+    if cls == ConcreteDriverCycleLeadtimeBased:
+        kwargs["leadtime"] = 6
+    driverobj = cls(**kwargs)
+    with raises(UWNotImplementedError) as e:
+        assert driverobj.output
+    assert str(e.value) == "The output() method is not yet implemented for this driver"
```

Comment:
`ConcreteDriverCycleBased` and `ConcreteDriverCycleLeadtimeBased` are inheriting the default, exception-raising `output` from `Driver`.